client/src/components/PitchDoc/ClaimPitchModal.tsx

```diff
@@ -0,0 +1,295 @@
+import React, {
+  FC,
+  ReactElement,
+  useState,
+  useEffect,
+  useCallback,
+} from 'react';
+import {
+  Modal,
+  Button,
+  Image,
+  Label,
+  Input,
+  Checkbox,
+  Popup,
+} from 'semantic-ui-react';
+import { IPitch } from 'ssw-common';
+
+import {
+  getOpenTeams,
+  updatePitchContributors,
+  updateClaimedPitches,
+  updatePitch,
+  loadUser,
+  isError,
+} from '../../utils/apiWrapper';
+import Homerun from '../../assets/homerun.svg';
+import Pfp from '../../assets/pfp.svg';
+import {
+  currentTeamsButtons,
+  teamToTeamsButtons,
+  enumToInterestButtons,
+  interestsButtons,
+} from '../../utils/constants';
+
+import PitchCard from './PitchCard';
+
+import '../../css/pitchDoc/ClaimPitchModal.css';
+
+interface IProps {
+  pitch: IPitch;
+  getAllUnclaimedPitches: () => Promise<void>;
+}
+
+const ClaimPitchModal: FC<IProps> = ({
+  pitch,
+  getAllUnclaimedPitches,
+}): ReactElement => {
+  const [firstOpen, setFirstOpen] = useState<boolean>(false);
+  const [secondOpen, setSecondOpen] = useState<boolean>(false);
+  const [openTeams, setOpenTeams] = useState<{
+    [key: string]: { current: number; target: number };
+  }>({});
+  const [selectedTeams, setSelectedTeams] = useState<string[]>([]);
+  const [pitchContributors, setPitchContributors] = useState<{
+    [key: string]: string;
+  }>({});
+  const [pitchLink, setPitchLink] = useState<string>('');
+  const [pitchAuthor, setPitchAuthor] = useState<string>('');
+  const [approvedBy, setApprovedBy] = useState<string>('');
+  const userId = '6031a866c70ec705736a79e5';
+  const temp_pfp = Pfp;
```

Comment:
Can you add some TODOs on line 61 or 62 to signify that we need to change this stuff